cds-snc/cppd-medical-report-sails
# MA View Application

# What this does

Adds the MA View for the medical report.

## To test

- Create a new report, filling in all the sections.
- Navigate to `/en/sessions` you will see a list of sessions with an option to Download or View
- Select View for the session you just created

## What's missing

- We are currently unable to calculate BMI because the data for it is not captured in a structured way. We can add this in future though. Currently just displaying a fake number.
- Links to download documents. We don't actually upload or store documents at this point, it's all just fake. So no links for now.
- We also don't have any authentication, nor a list screen for all the incoming medical reports, so I just repurposed the sessions route for now.
